ports/stm32/rfcore.c

```diff
@@ -68,9 +68,7 @@
 #define HCI_EVENT_COMMAND_COMPLETE     (0x0E) // <num packets><opcode 16><status><data...>
 #define HCI_EVENT_COMMAND_STATUS       (0x0F) // <status><num_packets><opcode 16>
 
-// There can be quite long delays during firmware update.
 #define SYS_ACK_TIMEOUT_MS (1000)
```

Comment:
maybe this can be decreased to previous value of 250?

Reply:
Yep, see above. Have made the FUS have its own override for the "installing" step, but now uses the 250ms default otherwise.